config/routes.rb

```diff
@@ -30,16 +30,13 @@
     namespace :v1 do
       resources :resources, except: [:new]
 
-      Brassica::Api.readable_models.map { |klass| klass.name.underscore.pluralize }.each do |model_name|
+      Api.readable_models.map { |klass| klass.name.underscore.pluralize }.each do |model_name|
```

Comment:
I think we want that for consistency in the API. Actual naming of db table does not matter.